idaes/models_extra/column_models/condenser.py

```diff
@@ -243,26 +243,25 @@ def build(self):
 
         self._make_ports()
 
+        @self.Expression(doc="Reflux calculations")
         def rule_reflux(self):
```

Comment:
You should give this a more indicative name as whatever you use here will be the name of the resulting `Expression` object.